Formula/spidermonkey@91.rb

```diff
@@ -0,0 +1,77 @@
+class SpidermonkeyAT91 < Formula
+  desc "JavaScript-C Engine"
+  homepage "https://spidermonkey.dev"
+  url "https://archive.mozilla.org/pub/firefox/releases/91.6.0esr/source/firefox-91.6.0esr.source.tar.xz"
+  version "91.6.0"
+  sha256 "7e802832152c39588b9a5c8392e90c1b00036bf948fa4a97a7af0d1435ba09a1"
+  license "MPL-2.0"
+
+  # Spidermonkey versions use the same versions as Firefox, so we simply check
+  # Firefox release versions.
+  livecheck do
+    url "https://www.mozilla.org/en-US/firefox/releases/"
+    regex(%r{href=.*?[^\d.](91(?:\.\d+)+)/releasenotes/?["' >]}i)
+  end
+
+  depends_on "autoconf@2.13" => :build
```

Comment:
There is also a bundled autoconf at `build/autoconf/autoconf.sh` if we want to consider deprecating `autoconf@2.13` formula in future. Fedora uses bundled 2.13 to avoid dependency https://src.fedoraproject.org/rpms/mozjs91/blob/rawhide/f/mozjs91.spec#_135